fdbserver/TLogInterface.h

```diff
@@ -51,6 +51,7 @@ struct TLogInterface {
 	RequestStream<struct TLogDisablePopRequest> disablePopRequest;
 	RequestStream<struct TLogEnablePopRequest> enablePopRequest;
 	RequestStream<struct TLogSnapRequest> snapRequest;
+	RequestStream<struct TLogUnblockPeekRequest> unblockPeekRequest;
```

Comment:
This is probably a protocol incompatible change, not sure if we can apply it to 7.1 branch.